contrib/presets/colors/protanopia-deuteranopia.conf

```diff
@@ -0,0 +1,105 @@
+# Preset for Red-green dichromacy color blindness.
+
+__lp_foreground_color 3
+YELLOW="${_LP_OPEN_ESC}${af_color}${_LP_CLOSE_ESC}"
+BOLD_YELLOW="${_LP_OPEN_ESC}${_LP_TI_BOLD-}${af_color}${_LP_CLOSE_ESC}"
+
+# Replaces reds by yellows.
+LP_COLOR_MARK_ROOT=${BOLD_YELLOW}
+LP_COLOR_NOWRITE=${YELLOW}
+LP_COLOR_COMMITS_BEHIND=${BOLD_YELLOW}
+LP_COLOR_CHANGES=${YELLOW}
+
+lp_terminal_format 226 -1 1 0 0 11
+LP_COLOR_TELNET=${lp_terminal_format}
+
+# fg, bg, bold, underline, fallback_fg, fallback_bg
+# -1 = none, -2 = previous, -3 = switch fg/g
+lp_terminal_format 46 -1 0 0 10 -1 # light green
+LP_CM_0="$lp_terminal_format"
+lp_terminal_format 40 -1 1 0 10 -1 # green
+LP_CM_1="$lp_terminal_format"
+lp_terminal_format 34 -1 1 1 10 -1 # dark green
+LP_CM_2="$lp_terminal_format"
+lp_terminal_format 70 -1 0 0 2 -1 # kaki
+LP_CM_3="$lp_terminal_format"
+lp_terminal_format 106 -1 1 0 2 -1 # yellowish kaki
+LP_CM_4="$lp_terminal_format"
+lp_terminal_format 142 -1 0 0 3 -1 # green brown
+LP_CM_5="$lp_terminal_format"
+lp_terminal_format 178 -1 1 0 3 -1 # brown orange
+LP_CM_6="$lp_terminal_format"
+lp_terminal_format 214 -1 0 0 11 -1 # orange
+LP_CM_7="$lp_terminal_format"
+lp_terminal_format 220 -1 1 0 11 -1 # yellow
+LP_CM_8="$lp_terminal_format"
+lp_terminal_format 226 -1 1 0 0 11 # light yellow
+LP_CM_9="$lp_terminal_format"
+
+LP_COLORMAP=( ${LP_COLORMAP[@]+"${LP_COLORMAP[@]}"} )
+if [[ ${#LP_COLORMAP[@]} == 0 ]]; then
+    LP_COLORMAP=(
+        "$LP_CM_0"
+        "$LP_CM_1"
+        "$LP_CM_2"
+        "$LP_CM_3"
+        "$LP_CM_4"
+        "$LP_CM_5"
+        "$LP_CM_6"
+        "$LP_CM_7"
+        "$LP_CM_8"
+        "$LP_CM_9"
+    )
+fi
+
+# Default Configuration (no reds).
+# LP_COLOR_CMAKE_DEBUG=${MAGENTA}
+# LP_COLOR_CMAKE_RWDI=${BLUE}
+# LP_COLOR_CMAKE_RELEASE=${CYAN}
+# LP_COLOR_PATH=${NO_COL}
+# lp_terminal_format 8 -1 0 0 -1
+# LP_COLOR_PATH_SEPARATOR=${lp_terminal_format}
+# LP_COLOR_PATH_SHORTENED=${lp_terminal_format}
+# lp_terminal_format -1 -1 1 0
+# LP_COLOR_PATH_VCS_ROOT=${lp_terminal_format}
+# LP_COLOR_PATH_LAST_DIR=${lp_terminal_format}
+# LP_COLOR_PATH_ROOT=${BOLD_YELLOW}
+# LP_COLOR_PROXY=${BOLD_BLUE}
+# LP_COLOR_ENV_VARS_UNSET=${BLUE}
+# LP_COLOR_ENV_VARS_SET=${BOLD_BLUE}
+# LP_COLOR_JOB_D=${YELLOW}
+# LP_COLOR_JOB_R=${BOLD_YELLOW}
+# LP_COLOR_JOB_Z=${BOLD_YELLOW}
+# LP_COLOR_ERR=${PURPLE}
+# LP_COLOR_ERR_MEANING=${LP_COLOR_ERR}
+# LP_COLOR_MARK=${BOLD}
```

Comment:
I'm not sure if this is something we can fix or not, but I loaded this preset after loading 256-colors-light.conf, and the mark became black. If I loaded this preset in a fresh env, it worked correctly.

Reply:
The right way to solve this is to reload defaults before loading a preset (or to have a way to unload a loaded preset somehow), which isn't possible with our current way of doing config. I would say probably not.